static/js/emoji.js

```diff
@@ -7,13 +7,20 @@ exports.emojis_by_name = {};
+
 emoji_names.push("zulip");
 
 _.each(emoji_names, function (value) {
     default_emojis.push({emoji_name: value, emoji_url: "/static/third/gemoji/images/emoji/" + value + ".png"});
 });
 
+_.each(unicode_emoji_names, function (value) {
+    default_emojis.push({emoji_name: value, emoji_url: "/static/third/gemoji/images/emoji/unicode/" + value + ".png"});
+});
```

Comment:
what's the thinking around adding the unicode emoji into `emojis_by_name` rather than adding a new `emojis_by_codepoint`?  Seems like the latter might be more clear...